tools/preproc/asm_file.cpp

```diff
@@ -26,33 +26,13 @@
 #include "char_util.h"
 #include "utf8.h"
 #include "string_parser.h"
+#include "../../include/constants/characters.h"
```

Comment:
It doesn't build on my end.
> asm_file.cpp:29:10: fatal error: ../../include/constants/characters.h: No such file or directory

Reply:
Yep. Forgot to commit that file. Fixed now